com.unity.netcode.gameobjects/Runtime/Serialization/ByteUnpacker.cs

```diff
@@ -356,11 +342,17 @@ public static unsafe void ReadValueBitPacked(FastBufferReader reader, out ushort
                     *ptr = *data;
```

Comment:
Small nit, can we go with something like

                    *(ptr++) = *(data++);
                    *(ptr++) = *(data++);
                    
to try to make it clear "we copy one item, we advance both pointers". If it gets refactored with either pointer starting early/later code update is easier. </minor>

Reply:
I see your point, but personally I think that's actually less clear about what's happen. I'd have to think about that to understand.

What about changing it to

```csharp
ptr[0] = data[0];
ptr[1] = data[1];
```

?

That's probably the clearest way to indicate that it's a mapping of bytes from `data` onto `ptr`

Reply:
Yep, that's clearer. In any case it is approved on my part, meaning: in the end, it's your call. :-)

Reply:
done